server/src/main/java/com/cloud/server/StatsCollector.java

```diff
@@ -1620,7 +1623,8 @@ protected void runInContext() {
                 for (StoragePoolVO pool : pools) {
                     List<VolumeVO> volumes = _volsDao.findByPoolId(pool.getId(), null);
                     for (VolumeVO volume : volumes) {
-                        if (volume.getFormat() != ImageFormat.QCOW2 && volume.getFormat() != ImageFormat.VHD && volume.getFormat() != ImageFormat.OVA && (volume.getFormat() != ImageFormat.RAW || pool.getPoolType() != Storage.StoragePoolType.PowerFlex)) {
+                        if (!List.of(ImageFormat.QCOW2, ImageFormat.VHD, ImageFormat.OVA).contains(volume.getFormat()) &&
```

Comment:
@rg9975, can you add the RAW format to the list?

Reply:
resolved in upcoming commit